src/components/pages/ImageClassification/ImageAnnotationModal.js

```diff
@@ -82,12 +33,29 @@ const LegendSquare = styled.div`
   border: ${({ color }) => `2px solid ${color}`};
 `
 
-const ImageAnnotationModal = ({ isModalDisplayed, setIsModalDisplayed }) => {
+const ImageAnnotationModal = ({ dataToReview, setDataToReview }) => {
   const mapContainer = useRef(null)
   const map = useRef(null)
+  const [dimensions, setDimensions] = useState()
+
+  const _getImageDimensions = useEffect(() => {
+    const imageForMap = new Image()
+    imageForMap.src = dataToReview.image
+    imageForMap.onload = () => {
+      const longerSide = Math.max(imageForMap.naturalWidth, imageForMap.naturalHeight)
+      const imageScale = longerSide > MAX_DIMENSION ? MAX_DIMENSION / longerSide : 1
+
+      setDimensions({
+        height: imageForMap.naturalHeight * imageScale,
+        width: imageForMap.naturalWidth * imageScale,
+        scale: imageScale,
+      })
+    }
+    // eslint-disable-next-line
+  }, [])
```

Comment:
don't we need `dataToReview.image` as a dep?

Reply:
could add error handling `imageForMap.onerror`

Reply:
yes i think closing the modal + toast is a good idea - and same for what i've implemented in for the image drop modal